app/components/settings/ChangePassword.tsx

```diff
@@ -67,19 +69,24 @@ const ChangePassword = () => {
     setVerifiedPassword(value);
   };
 
+  const handleCurrentPassword = ({ value }: { value: string }) => {
+    setCurrentPassword(value);
+  };
+
   const startUpdatingPassword = () => setIsEditMode(true);
 
-  const clear = () => {
-    setNewPassword('');
-    setVerifiedPassword('');
-    setNewPasswordError(null);
+  const isCurrentPasswordValid = async () => {
+    // @ts-ignore
```

Comment:
Please, do not use `// @ts-ignore`
Most likely you need to explicitly set the proper type to `dispatch` and the problem will go away:
```
const dispatch: AppThDispatch = useDispatch();
```